projects/kit/components/tabs/tabs-with-more/tabs-with-more.component.ts

```diff
@@ -58,8 +58,10 @@ export class TuiTabsWithMoreComponent implements AfterViewInit {
     @HostBinding('class._underline')
     underline = this.options.underline;
 
-    @Input()
-    activeItemIndex = 0;
+    @Input('activeItemIndex')
+    set itemIndex(index: number) {
+        this.updateActiveItemIndex(index);
```

Comment:
This method emits back "changes" output. Would it cause any issues?

Reply:
maybe 
```
this.activeItemIndex = activeItemIndex;
this.maxIndex = this.getMaxIndex();
```
?

Reply:
fixed